client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java

```diff
@@ -254,38 +258,42 @@ public ControllerImpl(ManagedChannelBuilder<?> channelBuilder) {
 
     @Override
     public CompletableFuture<Boolean> scaleStream(final Stream stream, final List<Integer> sealedSegments,
+                                                  final Map<Double, Double> newKeyRanges, final long timeoutMillis,
```

Comment:
I actually take it back. A long poll is a read operation, and that I suggested is a mix of "write" (trigger scale) and "read" (wait for outcome). If we want to have a long poll, it should be just a read operation that's idempotent and can be applied any time.